api/ruby/app.rb

```diff
@@ -18,7 +18,7 @@
 
 enable :logging
 
-client = Recurly::Client.new(api_key: ENV['RECURLY_API_KEY'])
+client = Recurly::Client.new(api_key: ENV['RECURLY_API_KEY'], region: :eu)
```

Comment:
I think it's fine for now, but eventually we might want to make this more dynamic so it could either be ":us" or ":eu".  You could maybe have the user select it on the welcome page.